subsys/net/ip/tcp.c

```diff
@@ -1035,9 +1053,24 @@ bool net_tcp_ack_received(struct net_context *ctx, u32_t ack)
 			continue;
 		}
 
-		seq = sys_get_be32(tcp_hdr->seq) + net_pkt_appdatalen(pkt) - 1;
+		seq_len = net_pkt_appdatalen(pkt);
+		/* Each of SYN and FIN flags are counted
```

Comment:
Please "open" these added lines a bit and add empty lines, now it looks very crowded.

Reply:
Done